sdk/core/azure-core/inc/azure/core/internal/strings.hpp

```diff
@@ -2,17 +2,17 @@
 // SPDX-License-Identifier: MIT
 
 /**
- * @brief Public utility functions for strings.
+ * @brief Internal utility functions for strings.
  *
  */
 #pragma once
 
 #include <string>
 
-namespace Azure { namespace Core { namespace Strings {
+namespace Azure { namespace Core { namespace Internal {
```

Comment:
I think it should be Azure::Core::Internal::Strings::

Why do we want to flatten the Internal API?   We can keep classifications inside the internal API

Reply:
Once this change is in (main goal is moving from public to internal), and we have moved it to Internal, we can add sub-namespaces and group them however we like. I don't want to preemptively add a sub-namespace until we see what other types are going to move. Any future PR that adds more to A::C::I can adjust them, though I am not sure we need to.

Reply:
Internal::Strings results in better looking code when using the methods.